README.md

```diff
@@ -1 +1,110 @@
-# predictive-market-cameligo
\ No newline at end of file
+# Betting Contract
+
+This contract represents a Tezos contract written in CameLIGO in which users can bet on events added by the Admin or an Oracle.
+
+The current implementation of the contract is as follows :
+
+## Structure :
+- a `Betting` contract, the main contract
+- _(optional)_ a `callback` contract for the `Betting`
+- _(optional)_ a `mock Oracle` contract
+- _(optional)_ a `callback` contract for the `mock Oracle`
+
+## Storage :
+```ocaml
```

Comment:
Also can you provide a document which specifies how to deploy the different pieces `contracts - bettting & oracle` & the `backend`

Reply:
Should we explicit the content of the initial storages in the `make deploy` command ?
Or should we have a separate section for each contract ?